src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/changes/beneficialowner/psc/CorporateBeneficialOwnerPsc.java

```diff
@@ -11,7 +11,7 @@ public class CorporateBeneficialOwnerPsc extends Psc {
   @JsonProperty("corporateName")
   private String corporateName;
   @JsonProperty("companyIdentification")
-  private CompanyIdentification companyIdentification;
+  private CompanyIdentification companyIdentification = new CompanyIdentification();
```

Comment:
Since we're not currently initialising objects in our models, can this be done in the service instead?